multiqc/utils/search_patterns.yaml

```diff
@@ -60,6 +60,8 @@ htseq:
     contents: '__too_low_aQual'
 hicup:
     fn: 'HiCUP_summary_report*'
+jellyfish:
+    fn: '*.hist'
```

Comment:
This is the only one I am not able to address.
So Jellyfish does not create a file with a fixed extension but the default extension is `.jf`. The `.jf` file  is the database file which also contains the keyword `jellyfish` that might be used in combination with the extension to find the right file
```
jellyfish:
   fn: ´*.jf´
   contents: 'jellyfish'
```

however I do not need this file but I need the one is generated by running jellyfish command
```
jellyfish histo -o file.hist -f file.jf
```

`file.hist` does not contain anything except the data to plot. The `.hist` extension is arbitrary but should be pretty natural.

A possibility might be that I look for `.jf` and jellyfish as key, and then I use the `fn` to see if the `hist` is present. If no I can terminate the execution or skip the sample without the `.hist` file. 
Not sure that this way of proceed agrees with MultiQC behaviour...